package.json

```diff
@@ -348,6 +348,13 @@
           "default": false,
           "markdownDescription": "Disable sending anonymous usage statistics to SonarSource. Click [here](https://github.com/SonarSource/sonarlint-vscode/blob/master/telemetry-sample.md) to see a sample of the data that are collected.",
           "scope": "window"
+        },
+        "sonarlint.cleanAsYouCode": {
+          "order": 100,
+          "type": "boolean",
+          "default": false,
+          "markdownDescription": "Enable SonarLint Clean as You Code mode. Click [here](https://docs.sonarsource.com/sonarqube/latest/user-guide/clean-as-you-code/) to learn more about Clean as You Code.",
```

Comment:
For consistency - we would usually just say `[Learn more](...) about Clean as You Code`